tests/TestCase/Mailer/EmailTest.php

```diff
@@ -2712,6 +2712,7 @@ public function testJsonSerialize()
         ];
         $this->assertEquals($expected, $result);
 
+        debug(unserialize(serialize($this->CakeEmail)));
         $result = json_decode(json_encode(unserialize(serialize($this->CakeEmail))), true);
```

Comment:
I think so. @jadb added this serialization feature to for eg. be able to serialize and store the email object in a queue to be later read by a worker and recreate the email object.